conditional/templates/spring_eval_slideshow.html

```diff
@@ -2,129 +2,32 @@
 {% block title %}
```

Comment:
Please remove this file from this PR as it's not relevant to the change being introduced and it's not complete.